index.html

```diff
@@ -5,672 +5,625 @@
-                        </article>
-
-<!-- toilet paper --><article class="course social">
-                            <a href="images/Marcy_Gooberman_Hello_Alfred_3.gif"
-                                class="shinybox"
-                                title="Hello Alfred">
-                                <img src="images/Marcy_Gooberman_Hello_Alfred_3.gif" alt="">
-                                Hello Alfred
-                            </a>
-                        </article>
-    
-<!-- oval office --><article class="course illustration motion">
-                            <a href="images/Marcy_Gooberman_Cosmo_Sex_4.gif"
-                                class="shinybox"
-                                title="Cosmopolitan.com">
-                                <img src="images/Marcy_Gooberman_Cosmo_Sex_4.gif" alt="">
-                                Cosmopolitan.com
-                            </a>  
-                        </article>
-
-          <!-- squat --><article class="course social motion illustration">
-                            <a href="images/Marcy_Gooberman_WW_1.gif"
-                                class="shinybox"
-                                title="Weight Watchers">
-                                <img src="images/Marcy_Gooberman_WW_1.gif" alt="">
-                                Weight Watchers
-                            </a>
-                        </article>
-
-        <!-- cardi b --><article class="course social">
-                            <a href="images/Marcy_Gooberman_tabu_7.png"
-                                class="shinybox"
-                                title="tabú">
-                                <img src="images/Marcy_Gooberman_tabu_7.png" alt="">
-                                tabú
-                            </a>
-                        </article>                        
-
-
-                                                        
-            </div>
-
- 
+    <nav id="filter" class="filters">
+
+        <ul class="filter-list" data-filter-group="subject">
+            <li><a href="javascript:void(0)" class="filter-item" data-filter=".branding">Branding</a></li>
+            <li><a href="javascript:void(0)" class="filter-item" data-filter=".social">Social Media</a></li>
+            <li><a href="javascript:void(0)" class="filter-item" data-filter=".illustration">Illustration</a></li>
+            <li><a href="javascript:void(0)" class="filter-item" data-filter=".print">Print</a></li>
+
+            <li><a href="javascript:void(0)" class="filter-item" data-filter=".motion">Motion Graphics</a></li>
+
+            <li><a href="javascript:void(0)" class="filter-item" data-filter=".web">Web</a></li>
+
+        </ul><!-- .subject -->
+
+    </nav><!-- .filters -->
+
+    <div class="courses">
+
+
+        <!-- period pandemic -->
+        <article class="course social">
+            <a href="images/Marcy_Gooberman_Blume_1.png" class="shinybox social" title="Blume">
```

Comment:
on each child `<a>`, I appended the filtering class(es) from `<article>`.
eg: article says `class="course social"` therefore I took `social` and added it on the `<a>` so it reads `class="shinybox social"`